sorted.js

```diff
@@ -0,0 +1,12 @@
+function checkIfSorted(array) {
+  if (array instanceof (Array) == false || array.length == 0) {
+    return `function works with not empty arrays only`
+  }
+  for (let i = 0; i < array.length; i++) {
+    if (typeof (array[i]) != 'number') { return 'function works with numbers'};
```

Comment:
please move return statement to the new line

Reply:
done